### DISCUSSION TITLE
port already in use when change appResyncPeriod

### DISCUSSION THREAD

**Discussion by michaelollifarel05** [argoproj/argo-cd#6480] [Q&A]
![Capture](https://user-images.githubusercontent.com/78914548/122156173-a6910100-ce92-11eb-836e-bde852540570.PNG)
i have installed HA version of argocd, but still have this issue

**Comment by jannfis** [comment]
Hi, this parameter is not intended to be changed at runtime. You will need to edit the manifest of the `StatefulSet` resource `argocd-application-controller` and add this parameter to the container's startup command. Upon change, the pod(s) of the application controller will restart with the option set.
